README.md

```diff
@@ -60,6 +60,24 @@ are assuming terminal/shell commands.
 5. If you get the message `Web application available at http://127.0.0.1:3000` then mojolicious is running and waiting
    for any requests.
 
+### Getting the standalone renderer code running
+
+1. Clone the repository with `git clone --recursive https://github.com/drdrew42/renderer`
```

Comment:
I recommend that you point this to https://github.com/openwebwork/renderer.  At this point those are the same in terms of the code.  I have been ensuring that.  As such the code still references Andrew's branches, but I will change that soon.

Reply:
True.  I was using the instructions on @drdrew42 renderer site.